edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
+  border-bottom: 3px solid $pearson-dark-blue-courseware;
+}
+/*
+div.xmodule_display.xmodule_SequenceModule .sequence-nav ol li button.active .icon::before {
+  content: "\f111";
+} */
 
-      margin: ($baseline/2);
-      width: auto;
-      color: $blue-d1; // WCAG 2.0 AA requirements
-      text-align: center;
+div.xmodule_display.xmodule_SequenceModule .sequence-nav {
+  margin: 20px auto 30px;
+}
 
-      &:hover,
-      &:focus {
-        background: $gray-l4;
-      }
-    }
+.content-wrapper > .container {
+  max-width: 1920px !important;
+  width: 100%;
+}
 
-    .flt-right {
-      @include float(right);
-    }
+#main > .container {
+  border: none !important;
+}
+
+.content-wrapper .page-content-container {
+  border: none !important;
+}
+
+nav.courseware {
+  border-bottom: 1px solid $pearson-grey-courseware;
+}
+
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button.focused,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button:hover,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button:active,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button.active,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button.focused,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button:hover,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button:active,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button.active {
+  background-color: transparent;
+}
+
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button.focused .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button:hover .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button:active .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav-button.active .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button.focused .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button:hover .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button:active .icon,
+div.xmodule_display.xmodule_SequenceModule .sequence-nav button.active .icon {
+  font-size: 110% !important;
+}
+
+/*
+
+div.xmodule_display.xmodule_VideoModule .video {
+  background: none;
+  margin: 15px 0;
+  padding: 0;
+  box-shadow: 4px 0px 20px 1px rgba(0,0,0,0.3);
+}  */
+
+div.xmodule_display.xmodule_SequenceModule .sequence-nav ol li {
+  border-top-color: $pearson-grey-courseware;
+  border-left-color: $inverse;
+  border-right-color: $inverse;
+}
+
+.xblock--drag-and-drop .feedback {
+  border: none !important;
+}
+
+div.course-wrapper .course-content .vert-mod .vert, .course-wrapper .courseware-results-wrapper .vert-mod .vert {
+  border: none !important;
+}
+
+.sponsors > li > a > img {
+  width: 50%;
+}
+
+.sponsors > li:first-child {
+  text-align: right;
+}
+
+.sponsors > li:last-child {
+  text-align: left;
+}
+
+//info-home
+div.home div.page-header-main{
+  font-family: "Times New Roman" !important;
```

Comment:
Can we get this value from a setting? @luismorenolopera

Reply:
no